doc/source/dace.rst

```diff
@@ -71,6 +71,14 @@ dace.serialize module
    :undoc-members:
    :show-inheritance:
 
+dace.sdfg module
```

Comment:
What's happening here?

Reply:
Documentation is missing for the SDFG module in the readthedocs website. It's not directly related to the PR.